docs/YAML_documents/sample_file.yaml

```diff
@@ -0,0 +1,19 @@
+test_1:
+  TEST_ID: unique_string
+  object_size:
+    start: 0Kib
+    end: 100Kib
+  result_duration: 0d0h0m0s
+  sessions_per_node: 1
+  tool: s3api
+  operation: bucket
+
+test_2:
+  TEST_ID: unique_string
+  object_size:
+    start: 0Kb
+    end: 100Kb
+  result_duration: 0d0h0m0s
+  sessions_per_node: 2
+  tool: s3api
```

Comment:
Added example for part range start & end
Check `config/s3/s3api/multipart_partcopy_random.yaml`

Reply:
added